pysteps/extrapolation/semilagrangian.py

```diff
@@ -16,6 +16,7 @@
 
 import numpy as np
 import scipy.ndimage.interpolation as ip
+from numpy.ma import MaskedArray
```

Comment:
do we need it here? I asssume you excludde all your changes in extrapolation, hence this import is not needed